### PR TITLE
fix version parser

### PR DESCRIPTION
**Pull request by Xxyuri2005xX**

### CODE REVIEW

File: bedwars-plugin/src/main/java/com/tomkeuper/bedwars/BedWars.java [hunk 1]
```diff
@@ -874,7 +874,7 @@ private boolean handleWorldAdapter() { //todo fix version check because current
         try {
             int major = Integer.parseInt(versionString[0]);
             int minor = Integer.parseInt(versionString[1]);
-            int release = versionString.length >= 3 ? Integer.parseInt(versionString[2]) : 0;
+            int release = versionString.length > 2 ? Integer.parseInt(versionString[2]) : 0;
```
Comment (tomkeuper) [inline bedwars-plugin/src/main/java/com/tomkeuper/bedwars/BedWars.java:877]: This check is basically the same. Greater or equal to 3 is the same as greater than 2

Comment (Xxyuri2005xX) [reply bedwars-plugin/src/main/java/com/tomkeuper/bedwars/BedWars.java:877]: it may be the same but this way it is more let's say clean

Comment (tomkeuper) [reply bedwars-plugin/src/main/java/com/tomkeuper/bedwars/BedWars.java:877]: But this doesnt fix anything, and in my opinion its exactly the same as before.

Comment (Xxyuri2005xX) [reply bedwars-plugin/src/main/java/com/tomkeuper/bedwars/BedWars.java:877]: whatever, it's your choice what to do

**Pull request closed without merging**